server/src/github.com/alexeyknyshev/tools/server_sqlite_to_tarantool/server_sqlite_to_tarantool.go

```diff
@@ -1197,16 +1197,77 @@ func migrateClustersGeo(tnt *tarantool.Connection, quadKeySet map[string][]uint3
 			log.Println("Error", err)
 			log.Println("Code", resp.Code)
 			log.Println("Data", resp.Data)
-			return;
+			return
 		}
 
 // 		log.Println(quadKey)
 	}
 }
 
+type Metro struct {
+	Id        uint32
+	Latitude  float64
+	Longitude float64
+	Town_id   uint32
+	Branch_id uint32
+	Name      string
+	Ext       string
+}
+
+func migrateMetro(townsDb *sql.DB, tnt *tarantool.Connection) {
+	SpaceId, err := getTntSpaceId(tnt, "metro")
+	if err != nil {
+		log.Fatalf("cannot get space id: %v", err)
+	}
+
+	err = tntSpaceClear(tnt, SpaceId)
+	if err != nil {
+		log.Fatalf("cannot drop space '%s': %v", "metro", err)
+	}
+
+	context := "migrateMetro"
+	var tupleCount int
+	err = townsDb.QueryRow(`SELECT COUNT(*) FROM metro`).Scan(&tupleCount)
+	if err != nil {
+		log.Fatalf("%s: metro: %v\n", context, err)
+	}
+	var metroT Metro
+	rows, err := townsDb.Query(`SELECT id, latitude, longitude, town_id, branch_id, name, ext FROM metro`)
+	if err != nil {
+		log.Fatalf("%s: %v\n", context, err)
+	}
+	currentMetroIndex := 0
+	for rows.Next() {
+		err = rows.Scan(&metroT.Id, &metroT.Latitude, &metroT.Longitude, &metroT.Town_id, &metroT.Branch_id, &metroT.Name, &metroT.Ext)
+		if err != nil {
+			log.Fatalf("%s: sql scan error: %v\n", err)
+			return
+		}
+
+		resp, err := tnt.Insert(SpaceId, []interface{}{
+			metroT.Id, []float64{metroT.Latitude, metroT.Longitude}, metroT.Town_id,
```

Comment:
Make sure that u inserting coordinate parts (longitude, latitude) in right order.
https://github.com/alexeyknyshev/android_cash_points/blob/master/server/src/github.com/alexeyknyshev/tools/server_sqlite_to_tarantool/server_sqlite_to_tarantool.go#L730
https://github.com/alexeyknyshev/android_cash_points/blob/master/server/src/github.com/alexeyknyshev/tools/server_sqlite_to_tarantool/server_sqlite_to_tarantool.go#L788